test/terra/backends/test_qasm_simulator.py

```diff
@@ -46,6 +46,7 @@
 from test.terra.backends.qasm_simulator.qasm_save import QasmSaveDataTests
 # Snapshot tests
 from test.terra.backends.qasm_simulator.qasm_snapshot import QasmSnapshotStatevectorTests
+from test.terra.backends.qasm_simulator.qasm_snapshot import QasmSnapshotStatevectorKetTests
```

Comment:
These can also be removed, and instead add tests to the `qasm_save_statevector.py` file (see the current statevector save examples). For tests you can compare the output to `qi.Statevector(circ).to_dict()`